tools/log_analyser/main.py

```diff
@@ -0,0 +1,7 @@
+from parser import log_parser
+from outputgenerator import generate_gsheet
+from inputreader.user_input import UserInput
+user_input_obj = UserInput()
+logs = user_input_obj.get_input()
+global_data = log_parser.general_parser(logs)
```

Comment:
please add documentation to code e.g. what each class represents, what each function does etc

Reply:
Added documentation